samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -349,16 +349,20 @@ Disabling key features
 
 The following key features of this sample may be independently disabled:
 
-* GNSS-based location tracking - by setting the :ref:`CONFIG_LOCATION_TRACKING_GNSS <CONFIG_LOCATION_TRACKING_GNSS>` option to disabled.
-* Cellular-based location tracking - by setting the :ref:`CONFIG_LOCATION_TRACKING_CELLULAR <CONFIG_LOCATION_TRACKING_CELLULAR>` option to disabled.
-* Wi-Fi-based location tracking - by setting the :ref:`CONFIG_LOCATION_TRACKING_WIFI <CONFIG_LOCATION_TRACKING_CELLULAR>` option to disabled.
-* Temperature tracking - by setting the :ref:`CONFIG_TEMP_TRACKING <CONFIG_TEMP_TRACKING>` option to disabled.
+* GNSS-based location tracking - by setting the :kconfig:option:`CONFIG_LOCATION_TRACKING_GNSS` option to disabled.
+* Cellular-based location tracking - by setting the :kconfig:option:`CONFIG_LOCATION_TRACKING_CELLULAR` option to disabled.
+* Wi-Fi-based location tracking - by setting the :kconfig:option:`CONFIG_LOCATION_TRACKING_WIFI` option to disabled.
+* Temperature tracking - by setting the :kconfig:option:`CONFIG_TEMP_TRACKING` option to disabled.
 * GNSS assistance (A-GPS) - by setting the :kconfig:option:`CONFIG_NRF_CLOUD_AGPS` option to disabled.
 * Predictive GNSS assistance (P-GPS) - by setting the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS` option to disabled.
 * FOTA when using MQTT - by setting the :kconfig:option:`CONFIG_NRF_CLOUD_FOTA` option to disabled.
-* FOTA when using CoAP - by setting the :kconfig:option:`NRF_CLOUD_COAP_FOTA` option to disabled.
+* FOTA when using CoAP - by setting the :kconfig:option:`COAP_FOTA` option to disabled.
+* Shadow handling when using CoAP - by setting the :kconfig:option:`COAP_SHADOW` option to disabled.
```

Comment:
```suggestion
* FOTA when using CoAP - by setting the :ref:`CONFIG_COAP_FOTA <CONFIG_COAP_FOTA>` option to disabled.
* Shadow handling when using CoAP - by setting the :ref:`CONFIG_COAP_SHADOW <CONFIG_COAP_SHADOW>` option to disabled.
```

Reply:
Why is this better?  I actually changed lines 352-355 to use :kconfig:option: instead, because I did not see any difference in appearance in the preview form of the docs from the longer :ref:`name <name>` form.

Reply:
I am also a bit confused, I'm the one who mentioned Pete should change it, because I thought we were spposed to be using :kconfig:option: for kconfigs, and all the options further down the list use it